src/rtctools_heat_network/esdl/esdl_heat_model.py

```diff
@@ -1205,8 +1210,17 @@ def convert_gas_demand(self, asset: Asset) -> Tuple[Type[GasDemand], MODIFIERS]:
         """
         assert asset.asset_type in {"GasDemand"}
 
+        id_mapping = asset.global_properties["carriers"][asset.in_ports[0].carrier.id][
+            "id_number_mapping"
+        ]
+
+        # hydrogen_specfic_energy = 20.0 / 1.0e6
```

Comment:
Should this line be commented out? Else delete or add comment that it should not be deleted due to ....

Reply:
updated